### PR TITLE
PR: Change default behavior of --show-console option

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes: #5495
- Deprecate --show-console
- Add hide console cli parameter

### CODE REVIEW

File: bootstrap.py [hunk 1]
```diff
@@ -154,9 +157,13 @@
 
 # --- Executing Spyder
 
-if not options.hide_console and os.name == 'nt':
-    print("0x. Enforcing parent console (Windows only)")
-    sys.argv.append("--show-console")  # Windows only: show parent console
+if options.show_console:
+    print("(Deprecated) --show console do nothing, now the default behavior "
```
Comment (ccordoba12) [inline bootstrap.py]: do -> does

**Comment by ccordoba12** [comment]
Nice! I left a minor comment, then I'll merge.

File: spyder/app/mainwindow.py [hunk 2]
```diff
@@ -3047,8 +3047,13 @@ def main():
     # otherwise, optparse won't be able to exit if --help option is passed
     options, args = get_options()
 
+    if options.show_console:
+        print("(Deprecated) --show console does nothing, now the default "
+              " behavior is to show the console, use --hide-console if you "
+              "want to hide it")
```
Comment (ccordoba12) [inline spyder/app/mainwindow.py]: Where this `print` is printed? I'd say is not necessary...

Comment (rlaverde) [reply spyder/app/mainwindow.py]: It's printed in the console where spyder --show-console is executed

Comment (ccordoba12) [reply spyder/app/mainwindow.py]: Ok, thanks for the clarification!

**Pull request merged**